definitions/json/State/State.json

```diff
@@ -106,5 +106,13 @@
     "Description": "Multiple Transferred",
     "DisplayOrder": 5,
     "TitleDisplay": "# Multiple Name: ${multipleName}\r\n# Multiple Number: ${multipleReference}"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "ID": "Accepted",
```

Comment:
Why create a new state? Is this required?

Reply:
Removed